upstream-community-operators/crossplane/0.9.0/crossplane.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,340 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crossplane.v0.9.0
+  namespace: crossplane-system
+  annotations:
+    capabilities: "Basic Install"
+    support: Community
+    categories: 'Application Runtime, Cloud Provider, Database, Integration &
+    Delivery, Networking, Storage'
+    containerImage: "crossplane/crossplane:v0.9.0"
+    createdAt: 2020-04-01T11:59:59Z
+    support: Crossplane Developers
```

Comment:
Super small nit: this property appears twice. Tip: use https://operatorhub.io/preview to get a preview of how your CSV would render on the side. There is a YAML parser included.